corona-warn-app/cwa-app-android
# Onboarding flow refactor (EXPOSUREAPP-2995)

## Description


This PR introduces the MVVM pattern with dependency injection for the onboarding flow. It also introduces the code skeleton for running instrumentation tests for each of the fragments in the onboarding flow.

## How to test

 Run the app and click around the onboarding flow. No behaviour changes should be noticeable. Make sure you delete the app's data before running if you've already been using the app, otherwise the onboarding flow won't show up.
